### PR TITLE
media-sound/gmusicbrowser improvements

### PR DESCRIPTION
**Pull request by bor**

### CODE REVIEW

**Comment by bor** [comment]
@SoapGentoo can you approve my request for proxied maintain?

**Comment by bor** [comment]
Also calling @gentoo-perl guys here, since there is one new perl module added.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+DEPEND="sys-devel/gettext"
+
+src_install() {
+	emake \
+		DOCS="AUTHORS NEWS README" \
+		DESTDIR="${D}" \
+		iconsdir="${D}/usr/share/icons/hicolor" \
+		install
+
+	use doc && dohtml layout_doc.html
```
Comment (SoapGentoo) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild]: `dohtml` is deprecated in `EAPI=6`. Instead, do the following:
```
use doc && local HTML_DOCS=( layout_doc.html )
einstalldocs
```
and then you can likely also nuke the `DOCS` argument from the `emake` command, as those files are part of the standard set.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	!gstreamer? ( !gstreamer-0? ( !mplayer? ( ${OTHER_DEPEND} ) ) )
+	extras? ( dev-perl/gnome2-wnck )
+	libnotify? ( dev-perl/Gtk2-Notify )"
+DEPEND="sys-devel/gettext"
+
+src_install() {
+	emake \
+		DOCS="AUTHORS NEWS README" \
+		DESTDIR="${D}" \
+		iconsdir="${D}/usr/share/icons/hicolor" \
```
Comment (SoapGentoo) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild]: double-slashing, do `iconsdir="${D%/}/usr/share/icons/hicolor" \` instead

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild (hunk 1 above)
Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild]: done

**Comment by SoapGentoo** [comment]
@akhuettel could you please sign off on the Perl stuff?

**Comment by monsieurp** [comment]
@kentfredric otherwise

**Comment by monsieurp** [comment]
@bor hi! you forgot to add the manifest for Glib-Object-Introspection. could you mend this?

**Comment by bor** [comment]
@monsieurp  ok I will add it.

Weird, found what's why I missed it :(
```
$ repoman fix

RepoMan scours the neighborhood...
>>> Creating Manifest for /home/store/projects/gentoo/dev-perl/Glib-Object-Introspection

Note: use --include-dev (-d) to check dependencies for 'dev' profiles

Note: type "repoman full" for a complete listing.
RepoMan sez: "If everyone were like you, I'd be out of business!"
$ ls
Glib-Object-Introspection-0.40.0.ebuild  metadata.xml
```

**Comment by bor** [comment]
I can't :(
Please help to understand why.
```
$ ebuild Glib-Object-Introspection-0.40.0.ebuild digest
Appending /home/store/projects/gentoo to PORTDIR_OVERLAY...
>>> Creating Manifest for /home/store/projects/gentoo/dev-perl/Glib-Object-Introspection
$ ls
Glib-Object-Introspection-0.40.0.ebuild  metadata.xml
```

**Comment by kentfredric** [comment]
`repoman manifest` ?

**Comment by bor** [comment]
the same :(

**Comment by kentfredric** [comment]
It doesn't appear to have worked for gmusicbrowser either, double check that.

If neither break, your config is borked and you're going to need to fire up strace and dig into why your config is borked.

**Comment by bor** [comment]
I have found the problem.
It was about EAPI=6 and required update for MODULE_* -> DIST_*

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	dbus? ( dev-perl/Net-DBus )
+	gstreamer? ( ${GSTREAMER_DEPEND} )
+	gstreamer-0? ( ${GSTREAMER0_DEPEND} )
+	mplayer? ( || ( ${MPLAYER_DEPEND} ${MPV_DEPEND} ) )
+	!gstreamer? ( !gstreamer-0? ( !mplayer? ( ${OTHER_DEPEND} ) ) )
+	extras? ( dev-perl/gnome2-wnck )
+	libnotify? ( dev-perl/Gtk2-Notify )"
+DEPEND="sys-devel/gettext"
+
+DOCS="AUTHORS NEWS README"
```
Comment (SoapGentoo) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild]: remove these, they are handled by the default set of `einstalldocs`

File: media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild [hunk 4]
```diff
@@ -31,27 +32,30 @@ RDEPEND="dev-lang/perl
...
+	gstreamer? ( ${GSTREAMER_DEPEND} )
+	gstreamer-0? ( ${GSTREAMER0_DEPEND} )
+	mplayer? ( || ( ${MPLAYER_DEPEND} ${MPV_DEPEND} ) )
+	!gstreamer? ( !gstreamer-0? ( !mplayer? ( ${OTHER_DEPEND} ) ) )
+	extras? ( dev-perl/gnome2-wnck )
+	libnotify? ( dev-perl/Gtk2-Notify )"
 DEPEND="sys-devel/gettext
 	doc? ( dev-perl/Text-Markdown )"
 
+DOCS="AUTHORS NEWS README"
```
Comment (SoapGentoo) [inline media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild]: same here

**Comment by SoapGentoo** [comment]
@kentfredric could you please have another look?

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild (hunk 3 above)
Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15-r2.ebuild]: removed

File: media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild (hunk 4 above)
Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild]: removed

**Comment by akhuettel** [comment]
Perl part LGTM, can't say much about the rest offhand

**Comment by bor** [comment]
Thank you, guys.

**Pull request closed without merging**